### ISSUE TITLE
List View and Block Library regions are not always present in the DOM

### ISSUE THREAD

**Issue by talldan** [WordPress/gutenberg#44311]
### Description

Mentioned to me by @alexstine, and I couldn't find an existing issue, so reporting it now.

The List View and Block Library regions are not always present in the DOM. This makes accessing them via the navigate regions shortcut difficult or impossible.

It'd be good to make them work more consistently, like the 'Editor Settings' region, where the container DOM element is always present. Possibly there should be a way to open the sidebars when they're closed but focus lands within the region, similar to the way the 'Publish Panel' can be opened in the post editor.

From a tech debt point of view, it might be worth consolidating these separate regions into one if a good semantic name can be given to it.

### Step-by-step reproduction instructions

1. Open one of the editors
2. Use the navigate region shortcut (<kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>N</kbd> or <kbd>Ctrl</kbd>+<kbd>`</kbd>)
3. Cycle through the regions and notice the List View or Block Library regions are never announced.

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes
